web/src/routes/TagDetails/TagDetailsTabs.tsx

```diff
@@ -20,6 +20,7 @@ function getTabIndex(tab: string) {
 export default function TagTabs(props: TagTabsProps) {
   const [activeTabKey, setActiveTabKey] = useState<TabIndex>(TabIndex.Details);
   const navigate = useNavigate();
+  const location = useLocation();
```

Comment:
`location` here doesn't seem to getting used in the PR.

Reply:
it's used in `src/routes/TagDetails/TagDetailsTabs.tsx:35` but not defined in the context which causes issues